mne/stats/tests/test_connectivity.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: Simplified BSD
+
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from mne.stats import combine_connectivity
+from mne.utils import requires_sklearn
+
+
+@requires_sklearn
+@pytest.mark.parametrize('ns', [
+    (1,),
+    (2,),
+    (1, 1),
+    (1, 2),
+    (2, 1),
+    (3, 4),
+    (1, 1, 1),
+    (1, 1, 2),
+    (3, 4, 5),
+])
+def test_connectivity_equiv(ns):
+    """Test connectivity equivalence for lattice connectivity."""
+    from sklearn.feature_extraction import grid_to_graph
+    sk_ns = ns if len(ns) > 1 else (ns + (1,))
+    conn_sk = grid_to_graph(*sk_ns).toarray()
```

Comment:
for future maintainability, could we have slightly more transparent variable names for `ns`, `sk_ns`, and `conn_sk`?